code/game/objects/items/keycard_doors.dm

```diff
@@ -0,0 +1,78 @@
+/obj/machinery/door/keycard
```

Comment:
keep the keycards and their doors in their own files. just make a file for new stuff youre introducing with the syndie mothership. ruins like hellfactory keep their keycards and doors in their own files

Reply:
or dont idk